subcommands/keys/tuf_utils.go

```diff
@@ -254,6 +254,12 @@ func FindSigner(keyid, pubkey string, creds OfflineCreds) (*TufSigner, error) {
 	return nil, fmt.Errorf("Can not find private key for: %s", pubkey)
 }
 
+func FindRootSigner(root *client.AtsTufRoot, creds OfflineCreds) (*TufSigner, error) {
```

Comment:
We've started getting requests to release fioctl with semantic versioning. If you want to start making all the helpers public - then you are signing up to maintain compatibility and help ensure our releases are numbered correctly. I think it might be easier to keep it private for simple helpers.

Reply:
Even if we go for semantic versioning; I'm not sure we ever have an intent to provide these functiona as a library to reuse anywhere.
If we do - I'd declare that explicitly by moving them to a package outside `subcommands` or even a separate github project.

Anyway, I'm hearing the request to make these utils private for the second time.
So, probably, my life should be easier if I change all functions in this package to private as a separate commit; and then go and make these new functions private too.
Are you OK with that?

Reply:
> If we do - I'd declare that explicitly by moving them to a package outside subcommands or even a separate github project.

That's just fighting the language. A separate commit is fine. Or leaving it as-is. I'm just noting that once it's released - future PRs require a little more thought.

Reply:
I did it and also extracted the "cleanup" commits into a separate PR, so that we can close that one faster:
https://github.com/foundriesio/fioctl/pull/196